library_generation/test/generate_pr_description_unit_tests.py

```diff
@@ -66,7 +67,7 @@ def test_get_commit_messages_with_same_current_and_baseline_returns_empty_messag
             ),
         )
 
-    def test_generate_pr_description_with_no_change_in_config(self):
+    def test_generate_pr_description_with_no_change_in_config__creates_no_file(self):
```

Comment:
```suggestion
    def test_generate_pr_description_with_no_change_in_config_creates_no_file(self):
```

Reply:
I removed these tests but renamed the ones in `test/cli` according to this suggestion (thanks!). I found [this article](https://g3doc.corp.google.com/eng/doc/devguide/testing/unit/best-practices.md?cl=head&polyglot=python#naming) as a confirmation for the right pattern in python.